src/Options.js

```diff
@@ -164,7 +165,7 @@ export class Options {
   }
 
   get sourceMaps() {
-    return this.sourceMaps_;
+    return this.lowResolutionSourceMap || this.sourceMaps_;
```

Comment:
Lets keep the options orthogonal, and ignore `lowResolutionSourceMap` if `sourceMaps` is not set.

Forgot that the `sourceMaps` option was not just a boolean, so definitely backing this line out.